src/api.ts

```diff
@@ -68,7 +68,9 @@ router.post('/msg', async (req, res) => {
     const address = getAddress(req.body.address);
     const env = 'livenet';
     let network = env === 'livenet' ? '1' : '4';
-    if (msg.type !== 'settings') network = await getSpaceNetwork(msg.space, env);
+    if (!req.body.data.types.Space && !msg.settings)
```

Comment:
The first condition would fail if it's personal sign isnt? I think we should change it to something like req.body?.data?.types?.Space